rpm/btrfs-balancer.spec

```diff
@@ -17,15 +21,27 @@ Requires:	libkeepalive-glib-tools >= 1.3.3
 %setup -q -n %{name}-%{version}
 
 %build
-
+%qmake5
 make %{?jobs:-j%jobs}
 
 %install
 rm -rf %{buildroot}
-%make_install
+%qmake5_install
 
 %files
 %defattr(-,root,root,-)
+/etc/dbus-1/system.d/*
+/usr/share/dbus-1/system-services/*
 /lib/systemd/system/*
 %{_sbindir}/*
 
+%package sbj-config
```

Comment:
This line is creating a separate device-specific package. I used project sailfish-snapshot for orientation. We're currently in a hurry to get the first implementation approved in time, so IMHO, if the config should be moved into another project that collects several device-specific config files, it can be done later at any time as well.